drive-cli.wiki/How-to-install-?.md

```diff
@@ -0,0 +1,5 @@
+## for usage  
+If you need to just use drive-cli directly, Drive-cli has a published pip package which you can use to get your installation done directly.
+
+## for development
+If you wish to contribute to the project/ modify it/build it locally you can setup development version of drive cli as well.
```

Comment:
It would be nice if you add screenshots of the development mode installation.
Like
1. Get your client_secret.json from Oauth. 
2. Enable Drive Api for the project.
3. Rename the client_secret.json to oauth.json and place it in the drive_cli directory.
and further steps.

Reply:
agreed, @ahegde3 screenshots would be great. We'll get you some bonus points as well for the same

Reply:
Uploaded.

Reply:
@ahegde3 what i meant was screenshot of the entire development mode process. I have listed in the above comment.

Reply:
@shivam1708   i have uploaded the screenshots of entire dev process.

Reply:
Sorry for the late reply. I just saw and it seems good.